.github/workflows/testing.yml

```diff
@@ -24,12 +24,20 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       fail-fast: false
-      max-parallel: 1
+      max-parallel: 2
```

Comment:
why would one choose serial vs parallel here?

Reply:
I'm actually not sure why we were opting to run serially before. Each test suite creates a specific index and operates on that currently, so risk of collisions or issues in that sense should be low.

I'm guessing you think we should be doing parallel here?

Reply:
The reason was to manage quota usage. The API key tied to this account can only create a certain number of indexes at one time. We can make it more parallel if we up the quota, which is tied to the sdk-node-testing project in our prod org.